lib/routes/objects.js

```diff
@@ -92,6 +95,10 @@ module.exports =
                 `Version ${version} not found in '${env}'`
               );
             }
+
+            // If a version is specified, the response can be
+            // cached for longer time since it's unlucky to change
```

Comment:
```suggestion
            // cached for longer time since it's unlikely to change
```

Reply:
I know something was not right here ah ah. Thanks